wiki/Community/Discord_servers/osu!_Medal_Hunters/en.md

```diff
@@ -0,0 +1,47 @@
+---
+tags:
+  - medals
+  - medal hunting
+---
+
+# osu! Medal Hunters Discord server
+
+::: Infobox
+
+<!-- lint ignore heading-increment -->
+
+#### osu! Medal Hunters
+
+![Server logo](img/logo.png "The Medal Hunter Discord server logo, designed by Tanza3D")
+
+|  |  |
+| :-- | :-- |
+| Invite link | <https://discord.gg/osumedalhunters> |
+| Website | <https://osekai.net/> |
+
+:::
+
+*Notice: While working together in order to find Hush-Hush medals is encouraged, finding them by yourself is as well. Be advised that the website and server contain solutions to Hush-Hush medals.*
```

Comment:
I think it's fine like that, but just wanted to clarify that the only place on server you can find solutions at is `#announcements` channel, and only for recent medals.

Reply:
you're really likely to run into them in bot commands and just generally across the server, as bathbot and other people may post solutions. best to cover the entire server with that I feel

Reply:
Hmm, makes sense!